### PR TITLE
Update content ui

### PR DESCRIPTION
**Pull request by katbow**
This updates the UI to only contain a single field, changes the schema for the posts table, and displays stories in /posts if the content field from the table contains #story

### CODE REVIEW

File: web/controllers/post_controller.ex [hunk 1]
```diff
@@ -25,7 +25,7 @@ defmodule Healthlocker.PostController do
   end
 
   def index(conn, _params) do
-    posts = Repo.all(Post)
+    posts = Repo.all(from p in Post, where: ilike(p.content, "%#story%"))
```
Comment (tonydaly) [inline web/controllers/post_controller.ex]: It'd be nice to move this query into the model, something like this: https://blog.drewolson.org/composable-queries-ecto/

Then we could test it independently there.

Also we should probably add a controller test, not sure if I finished those before.

Comment (katbow) [inline web/controllers/post_controller.ex]: I did most of the controller tests last night, will finish them up this morning.

Comment (tonydaly) [inline web/controllers/post_controller.ex]: @katbow nice one, they look good.

Comment (katbow) [inline web/controllers/post_controller.ex]: > It'd be nice to move this query into the model, something like this: https://blog.drewolson.org/composable-queries-ecto/

I read through the post and I understand how to do it, but can you explain what the benefit is of doing it that one? Is it easier to write tests, or can the tests be better in some way?

Comment (tonydaly) [inline web/controllers/post_controller.ex]: For instance the controller test we have (which is good) doesn't really test whether this query works or not? By moving it into the Model is keeps the responsibility for querying in the model, and we can independently test it there e.g. create a number of posts some with the correct tag, and some without, and then tests that the results returned are what we expect.

**Comment by nelsonic** [comment]
@katbow this is looking good... 👍 
Please remove `in-progess` and `awaiting-review` when you feel it's in a "dev done" state. _thanks!_

File: test/controllers/tip_controller_test.exs [hunk 2]
```diff
@@ -1,9 +1,10 @@
 defmodule Healthlocker.TipControllerTest do
   use Healthlocker.ConnCase
+  alias Healthlocker.Post
+  import Healthlocker.Fixtures
 
-  test "lists all entries on index", %{conn: conn} do
+  test "successfully loads tips index", %{conn: conn} do
```
Comment (nelsonic) [inline test/controllers/tip_controller_test.exs]: prefer a _mix_ of the old test description with the new one ... something like:
`"index lists all tips"`

Comment (tonydaly) [inline test/controllers/tip_controller_test.exs]: My style for these tests:

```
describe "GET index" do
  test "lists all tips" do
    blah blah
  end

  test "returns 200" do
    assert conn.status == 200
  end
end
```

I'm a big fan of `describe` and `context` blocks within tests, it makes them a lot more readable in the tests and the output. Maybe I'll do a lil PR to demonstrate this.

Comment (nelsonic) [inline test/controllers/tip_controller_test.exs]: @tonydaly agreed. 👍  (_better syntax_)

**Pull request merged**